pkg/kubectl/resource_printer.go

```diff
@@ -85,6 +82,10 @@ func GetPrinter(format, formatArgument string) (ResourcePrinter, bool, error) {
 type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
 	PrintObj(runtime.Object, io.Writer) error
+
+	// Adds an ObjectProcessor. All object processors will be executed
+	// in the order they were added to the ResourcePrinter.
+	AddObjectPreprocessor(ObjectPreprocessor)
```

Comment:
I'd like a method to get back the current preprocessors.  Can we name them as well, so I can recognize what's in the chain?